packages/icons/icons.json

```
@@ -1332,42 +1332,42 @@
     {
         "displayName": "Duplicate",
         "iconName": "duplicate",
-        "tags": "copy, square, two",
+        "tags": "copy, square, two, copies, stack, photocopy, boxes, documents, multiple pages, square stack, depth order",
         "group": "action",
         "codepoint": 61843
     },
     {
         "displayName": "Ban circle",
         "iconName": "ban-circle",
-        "tags": "circle, refusal",
+        "tags": "circle, refusal, do not enter, inaccessible, screw head, pause, slow, null, subtract, minus, delete",
```

```suggestion
        "tags": "circle, refusal, do not enter, inaccessible, screw head, flathead, slow, null, subtract, minus, delete",
```

not sure about "pause" in this context

---

agreed